packages/esm-patient-search-app/src/types/index.ts

```diff
@@ -3,6 +3,7 @@ export interface SearchedPatient {
   uuid: string;
   identifiers: Array<Identifier>;
   person: {
+    [x: string]: any;
```

Comment:
It would be best to just add the attributes we need, especially as person only has so many attributes...